pyproject.toml

```diff
@@ -40,6 +43,18 @@ docs = [
 testing = [
     "ops-scenario>=7.0.5,<8",
 ]
+tracing = [
+    "otlp-json>=0.9.2",
```

Comment:
Nice! Do you want any review on this repo?

Reply:
Yes please!

It's a single file: https://github.com/dimaqq/otlp-json/blob/main/otlp_json/__init__.py